app/assets/stylesheets/global/african_poetics.scss

```diff
@@ -74,7 +74,7 @@ a.btn, .btn, input {
 }
 
 #header {
-  background-color: $brand-info;
+  background-color: #357fa0;
```

Comment:
still going to merge, but generally let's try to avoid hex codes in the scss - we should define in variables or use color mixing to get new color.